### PR TITLE
Ear 1033 validate other label radio and checkbox

### PR DESCRIPTION
**Pull request by zimboflyman**
Finally....

### What is the context of this PR?

**Acceptance criteria (user perspective)**
**_Empty "Other label" field_**

GIVEN I've added an "Other" option on a radio answer
WHEN the "Other label" field is empty
THEN I see a validation message on the "Other label" field which reads "Enter a label"

GIVEN I've added an "Other" option on a checkbox answer
WHEN the "Other label" field is empty
THEN I see a validation message on the "Other label" field which reads "Enter a label"

_**Not empty "Other label" field**_

GIVEN I've added an "Other" option on a radio answer
WHEN the "Other label" field is not empty
THEN I see no validation message on the "Other label" field

GIVEN I've added an "Other" option on a checkbox answer
WHEN the "Other label" field is not empty
THEN I see no validation message on the "Other label" field

### How to review

create a checkbox answer and select "add checkbox - other option" to add a checkbox with other option
make sure the additional 'other label' has working validation with correct message

do the same for Radio answer

### CODE REVIEW

File: eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/MultipleChoiceAnswer.test.js [hunk 1]
```diff
@@ -125,27 +125,27 @@ describe("MultipleChoiceAnswer", () => {
...
+
+  //   const Options = wrapper.find(Reorder).renderProp("children")(
+  //     extraProps,
+  //     option
+  //   );
+
+  //   expect(
+  //     Options.find("Field__StyledField")
+  //   ).toHaveLength(1);
+  // });
```
Comment (MattElcock) [inline eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/MultipleChoiceAnswer.test.js:148]:
```suggestion
```

**Review by MattElcock — CHANGES_REQUESTED** [review]

**Comment by MattElcock** [comment]
image

The distance between the error message and the bottom of the component needs to be increased a smidge, I think.

File: eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/MultipleChoiceAnswer.test.js (hunk 1 above)
Comment (zimboflyman) [reply eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/MultipleChoiceAnswer.test.js]: woops - now sorted

**Review by MattElcock — DISMISSED** [review]
Hi Shane, well done on this. Thanks, too, for modernising the class component. It's good to see that we're making these improvements when we come across them.

Everything looks nice, tidy, and works as expected. :)

**Review by farres1 — COMMENTED** [review]
The validation is really good!

I agree with Matt that the error message should be slightly higher. 

Also, the validation error message for unique radio/checkboxes no longer works. The error badge is still displayed, but the error message is not shown.

File: .gitignore [hunk 2]
```diff
@@ -35,6 +35,8 @@ graphql.config.json
 public/status.json
 .env
 .env.*local
+env.test
+env.*local
```
Comment (tbaggaley) [inline .gitignore:39]:
```suggestion
```

Haven't looked properly yet but can we get rid of these from the `.gitignore`? :-)

**Review by tbaggaley — CHANGES_REQUESTED** [review]

**Review by farres1 — APPROVED** [review]

**Review by MattElcock — APPROVED** [review]

**Pull request merged**